data-migration-route.md

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
Please update the TOC according to [docs-cn TOC](https://raw.githubusercontent.com/pingcap/docs-cn/master/TOC.md). @toutdesuite

Reply:
update in 52f3c9142cbe360f7838a5685c90538de13771fb